common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -117,6 +118,7 @@
   public static final String CONTENT_ID = "contentId";
   public static final String CONTENT_IDS = "contentIds";
   public static final String CONTENT_LIST = "contentList";
+  public static final String CONTENT_STORE_API_KEY = "content_store_api_key";
```

Comment:
Why a new variable is required? Can't existing env SUNBIRD_AUTHORIZATION (sunbird_authorization) be used?